entities/storobj/storage_object.go

```diff
@@ -208,6 +209,7 @@ func FromBinaryOptional(data []byte,
 		if err != nil {
 			return nil, err
 		}
+		// TODO vectors here has ALL the vectors, not just the named ones in the args
 		ko.Vectors = vectors
```

Comment:
`unmarshalTargetVectors` always unmarshals ALL the target vectors, not only those named in `addProps`. Should I modify this code to only unmarshal the named vectors or will that break callers?

Reply:
no, let's leave the behavior for target vectors as it stands